#! /bin/sh /usr/share/dpatch/dpatch-run
## 50_system_doc-2.9.dpatch by Thomas Weber <thomas.weber.mail@gmail.com>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: The example in system() doesn't match the described behaviour. The
## behaviour changed in 2.9.5

@DPATCH@
diff -urNad octave2.9-2.9.5~/src/toplev.cc octave2.9-2.9.5/src/toplev.cc
--- octave2.9-2.9.5~/src/toplev.cc	2006-03-15 22:27:35.000000000 +0100
+++ octave2.9-2.9.5/src/toplev.cc	2006-04-25 22:45:45.000000000 +0200
@@ -422,7 +422,7 @@
 command that was written to the standard output stream.  For example,\n\
 \n\
 @example\n\
-[output, status] = system (\"echo foo; exit 2\");\n\
+[status, output] = system (\"echo foo; exit 2\");\n\
 @end example\n\
 \n\
 @noindent\n\
